src/os/bluestore/FreelistManager.h

```diff
@@ -52,9 +52,11 @@ class FreelistManager {
   virtual void get_meta(uint64_t target_size,
     std::vector<std::pair<string, string>>*) const = 0;
 
-  virtual std::vector<zone_state_t> get_zone_states(KeyValueDB *kvdb) const {
+  virtual std::vector<zone_state_t> zoned_get_zone_states(KeyValueDB *kvdb) const {
```

Comment:
Use dynamic_cast for these methods as well?

Reply:
Done.